Author: Andreas Beckmann <anbe@debian.org>
Description: adjust some paths in the documentation according to fhs-compliant choices in Debian

--- a/sendmail/sendmail.8
+++ b/sendmail/sendmail.8
@@ -728,7 +728,7 @@ configuration file
  /etc/mail/helpfile
 help file
 .TP
- /etc/mail/statistics
+ /var/lib/sendmail/sendmail.st
 collected statistics
 .TP
  /var/spool/mqueue/*
--- a/smrsh/README
+++ b/smrsh/README
@@ -49,22 +49,22 @@ the bin directory used by smrsh.
 
 -DSMRSH_PATH=\"path\"	\"/bin:/usr/bin:/usr/ucb\"	The default search
 							path.
--DSMRSH_CMDDIR=\"dir\"	\"/usr/adm/sm.bin\"		The default smrsh
+-DSMRSH_CMDDIR=\"dir\"	\"/etc/mail/smrsh\"		The default smrsh
 							program directory
  
 These can be added to the devtools/Site/site.config.m4 file using the
 global M4 macro confENVDEF or the smrsh specific M4 macro
 conf_smrsh_ENVDEF.
 
-As root, install smrsh in /usr/libexec.  Using the Build script:
+As root, install smrsh in /usr/libexec/sendmail/.  Using the Build script:
 
 	host.domain# sh ./Build install
 
-For manual installation: install smrsh in the /usr/libexec
+For manual installation: install smrsh in the /usr/libexec/sendmail/
 directory, with mode 511.
 
-	host.domain# mv smrsh /usr/libexec
-	host.domain# chmod 511 /usr/libexec/smrsh
+	host.domain# mv smrsh /usr/libexec/sendmail/
+	host.domain# chmod 511 /usr/libexec/sendmail/smrsh
 
 
 
@@ -79,14 +79,14 @@ to run.  This list of allowable commands
 
 See the man page for aliases(5) if you are unfamiliar with the format of
 these specifications. Additionally, you should include in the list,
-popular commands such as /usr/ucb/vacation.
+popular commands such as /usr/libexec/sendmail/vacation.sendmail.
 
 You should NOT include interpreter programs such as sh(1), csh(1),
 perl(1), uudecode(1) or the stream editor sed(1) in your list of
 acceptable commands.
 
 If your platform doesn't have a default SMRSH_CMDDIR setting, you will
-next need to create the directory /usr/adm/sm.bin and populate
+next need to create the directory /etc/mail/smrsh and populate
 it with the programs that your site feels are allowable for sendmail
 to execute.   This directory is explicitly specified in the source
 code for smrsh, so changing this directory must be accompanied with
@@ -95,22 +95,22 @@ a change in smrsh.c.
 
 You will have to be root to make these modifications.
 
-After creating the /usr/adm/sm.bin directory, either copy the programs
+After creating the /etc/mail/smrsh directory, either copy the programs
 to the directory, or establish links to the allowable programs from
-/usr/adm/sm.bin.  Change the file permissions, so that these programs
+/etc/mail/smrsh.  Change the file permissions, so that these programs
 can not be modified by non-root users.  If you use links, you should
 ensure that the target programs are not modifiable.
 
 To allow the popular vacation(1) program by creating a link in the
-/usr/adm/sm.bin directory, you should:
+/etc/mail/smrsh directory, you should:
 
-	host.domain# cd /usr/adm/sm.bin
-	host.domain# ln -s /usr/ucb/vacation vacation
+	host.domain# cd /etc/mail/smrsh
+	host.domain# ln -s /usr/libexec/sendmail/vacation.sendmail vacation
 
 
 
 
-After populating the /usr/adm/sm.bin directory, you can now configure
+After populating the /etc/mail/smrsh directory, you can now configure
 sendmail to use the restricted shell.  Save the current sendmail.cf
 file prior to modifying it, as a prudent precaution.
 
@@ -125,7 +125,7 @@ help to locate it.
 
 In order to configure sendmail to use smrsh, you must modify the Mprog
 definition in the sendmail.cf file, by replacing the /bin/sh specification
-with /usr/libexec/smrsh.
+with /usr/libexec/sendmail/smrsh.
 
 As an example:
 
@@ -133,14 +133,14 @@ In most Sun Microsystems' sendmail.cf fi
 Mprog,	P=/bin/sh,   F=lsDFMeuP,  S=10, R=20, A=sh -c $u
 
 which should be changed to:
-Mprog,	P=/usr/libexec/smrsh,   F=lsDFMeuP,  S=10, R=20, A=sh -c $u
-          ^^^^^^^^^^^^^^^^^^
+Mprog,	P=/usr/libexec/sendmail/smrsh,   F=lsDFMeuP,  S=10, R=20, A=sh -c $u
+          ^^^^^^^^^^^^^^^^^^^^^^
 
 A more generic line may be:
 Mprog,		P=/bin/sh, F=lsDFM, A=sh -c $u
 
 and should be changed to;
-Mprog,		P=/usr/libexec/smrsh, F=lsDFM, A=sh -c $u
+Mprog,		P=/usr/libexec/sendmail/smrsh, F=lsDFM, A=sh -c $u
 
 
 After modifying the Mprog definition in the sendmail.cf file, if a frozen
--- a/smrsh/smrsh.8
+++ b/smrsh/smrsh.8
@@ -39,7 +39,7 @@ Briefly,
 .I smrsh
 limits programs to be in a single directory,
 by default
-/usr/adm/sm.bin,
+/etc/mail/smrsh,
 allowing the system administrator to choose the set of acceptable commands,
 and to the shell builtin commands ``exec'', ``exit'', and ``echo''.
 It also rejects any commands with the characters
@@ -56,7 +56,7 @@ so forwarding to ``/usr/ucb/vacation'',
 and
 ``vacation''
 all actually forward to
-``/usr/adm/sm.bin/vacation''.
+``/etc/mail/smrsh/vacation''.
 .PP
 System administrators should be conservative about populating
 the sm.bin directory.
@@ -86,9 +86,9 @@ to adjust the default search path
 (defaults to ``/bin:/usr/bin:/usr/ucb'')
 and/or \-DSMRSH_CMDDIR=\e"\fIdir\fP\e"
 to change the default program directory
-(defaults to ``/usr/adm/sm.bin'').
+(defaults to ``/etc/mail/smrsh'').
 .SH FILES
-/usr/adm/sm.bin \- default directory for restricted programs on most OSs
+/etc/mail/smrsh \- default directory for restricted programs on most OSs
 .PP
 /var/adm/sm.bin \- directory for restricted programs on HP UX and Solaris
 .PP
